src/recipe.rs

```diff
@@ -479,7 +483,7 @@ impl<'src, D: Display> ColorDisplay for Recipe<'src, D> {
       writeln!(f, "# {doc}")?;
     }
 
-    for attribute in &self.attributes {
+    for attribute in self.attributes.iter() {
```

Comment:
We should implement `IntoIterator` for `&AttributeSet` so we can avoid needing to call `.iter()`.